scripts/tasks.sh

```diff
@@ -490,32 +490,32 @@ spec:
   template:
     spec:
       tolerations:
-        - key: node-role.kubernetes.io/master
+        - key: node-role.kubernetes.io/control-plane
           operator: Exists
       containers:
         - name: rook-ceph-operator
           env:
             - name: DISCOVER_TOLERATION_KEY
-              value: node-role.kubernetes.io/master
+              value: node-role.kubernetes.io/control-plane
             - name: CSI_PROVISIONER_TOLERATIONS
               value: |
-                - key: node-role.kubernetes.io/master
+                - key: node-role.kubernetes.io/control-plane
                   operator: Exists
             - name: CSI_PLUGIN_TOLERATIONS
               value: |
-                - key: node-role.kubernetes.io/master
+                - key: node-role.kubernetes.io/control-plane
                   operator: Exists
 EOF
     fi
 
     # EKCO tolerations
     if kubernetes_resource_exists kurl deployment ekc-operator; then
-        kubectl -n kurl patch deployment ekc-operator --type=merge -p '{"spec":{"template":{"spec":{"tolerations":[{"key":"node-role.kubernetes.io/master","operator":"Exists"}]}}}}'
+        kubectl -n kurl patch deployment ekc-operator --type=merge -p '{"spec":{"template":{"spec":{"tolerations":[{"key":"node-role.kubernetes.io/control-plane","operator":"Exists"}]}}}}'
     fi
 
 
     # Taint all primaries
-    kubectl taint nodes --overwrite --selector=node-role.kubernetes.io/master node-role.kubernetes.io/master=:NoSchedule
+    kubectl taint nodes --overwrite --selector=node-role.kubernetes.io/control-plane node-role.kubernetes.io/control-plane=:NoSchedule
```

Comment:
Would it be possible to detect the kubernetes version and apply the proper taint for that version?